packages/mermaid/src/diagrams/gantt/ganttRenderer.js

```diff
@@ -572,7 +580,30 @@ export const draw = function (text, id, version, diagObj) {
           bottomXAxis.ticks(timeDay.every(every));
           break;
         case 'week':
-          bottomXAxis.ticks(timeWeek.every(every));
+          switch (weekday) {
```

Comment:
This switch-case statement could be refactored (as the code in all `case` clauses are quite similar), but this is not mandatory

Reply:
I think maps are the most suitable approach here, although I'm still fine with the current version. Maybe modify it later on?